contracts/examples/hello-world/tests/hello-world.ts

```diff
@@ -32,54 +32,53 @@ describe('hello-world', () => {
       decimals,
       granularity,
       liveLength,
-    {
-      accounts: {
-        feed: feed.publicKey,
-        authority: owner.publicKey,
-      },
-      signers: [feed],
-      preInstructions: [
-        await storeProgram.account.transmissions.createInstruction(feed, 8+192+6*24),
-      ],
-    });
-
-    await storeProgram.rpc.setWriter(
-      owner.publicKey,
       {
         accounts: {
           feed: feed.publicKey,
-          owner: owner.publicKey,
           authority: owner.publicKey,
         },
-      });
-    
+        signers: [feed],
+        preInstructions: [
+          await storeProgram.account.transmissions.createInstruction(
+            feed,
+            8 + 192 + 6 * 24
```

Comment:
Let's define these as descriptive constants in a single file and import them here